docs/2.guide/3.going-further/7.layers.md

```diff
@@ -104,6 +104,11 @@ If you want to extend a private remote source, you need to add the environment v
 If you want to extend a remote source from a self-hosted GitHub or GitLab instance, you need to supply its URL with the `GIGET_GITHUB_URL=<url>` or `GIGET_GITLAB_URL=<url>` environment variable - or directly configure it with [the `auth` option](https://github.com/unjs/c12#extending-config-layer-from-remote-sources) in your `nuxt.config`.
 ::
 
+::warning
++Nuxt Layers installed with Giget should only require Nuxt modules. Other dependencies cannot be resolved because they are installed in a nested structure (node_modules/.c12/layer_name/node_modules) rather than at the root level.
++To share dependencies between your layer and the Nuxt app, install your Nuxt Layer using a package manager instead of Giget.
```

Comment:
you should be able to share dependencies between your layer and the nuxt app.

I thought your issue was about sharing dependencies between eslint and the layer?

Reply:
Oh, I think I misunderstood you.
This is quite complex !

I guess I thought the dependencies wouldn't work because of this error in VS Code, but now I see that this is probably also an ESLint error, since as stated previously, the `twMerge` function works even with the error.
<img width="1096" alt="391588409-572f6ca4-90dc-4530-b5ba-b895384a6567" src="https://github.com/user-attachments/assets/5c59aed1-1066-447a-8597-e4accaa8a804">

So if I understand properly, dependencies installed by giget will work if they are required by the nuxt app, but won't work if they are only required outside the nuxt app because Nuxt do some magic behind the scene with the include/exclude (For exemple, `.nuxt/tsconfig.json`), but can't do it for dependencies that are required outside of Nuxt ?